// api/mockWatches.js
export default[
    {
    id: "1",
    categoryId: "1",
    name: "ROYAL LONDON 41383-05",
    brand: 'Royal London',
    price: 535,
    image: "/uploads/41383-05-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "42 мм",
    glass: "минеральное",
    beltType: "натуральная кожа",
    faceColor: "белый",
    beltColor: "коричневый",
    waterProtection: "WR50-5bar",
    warranty: "2 года",
    },
    {
    id: "10",
    categoryId: "1",
    name: "ROYAL LONDON 41334-04",
    brand: 'Royal London',
    price: 322,
    image: "/uploads/41334-04-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "44 мм",
    glass: "минеральное",
    beltType: "натуральная кожа",
    faceColor: "белый",
    beltColor: "коричневый",
    waterProtection: "WR50",
    warranty: "2 года",
    },
    {
    id: "3",
    categoryId: "2",
    name: "JACQUES LEMANS 1-1859H",
    brand: 'Jacques Lemans',
    price: 560,
    image: "/uploads/1-1859H-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "40 мм",
    glass: "минеральное",
    beltType: "нержавеющая сталь",
    faceColor: "стальной",
    beltColor: "стальной",
    waterProtection: "WR100 - 10bar",
    warranty: "2 года",
    },
    {
    id: "9",
    categoryId: "2",
    name: "JACQUES LEMANS U-58B",
    brand: 'Jacques Lemans',
    price: 560,
    image: "/uploads/948db24cf05811e781ad0050569f2b64_5fac3ba20bb411e881af0050569f2b64-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "40 мм",
    glass: "минеральное",
    beltType: "натуральная кожа",
    faceColor: "стальной",
    beltColor: "черный",
    waterProtection: "WR100 - 10bar",
    warranty: "2 года",
    },
    {
    id: "5",
    categoryId: "3",
    name: "CASIO EFS-S530L-2A",
    brand: 'Casio',
    price: 525,
    image: "/uploads/a76590c7b1e111e881cb0050569f2b64_7da7dcd1baf711e881cb0050569f2b64-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "40 мм",
    glass: "сапфировое",
    beltType: "натуральная кожа",
    faceColor: "синий",
    beltColor: "синий",
    waterProtection: "WR100 - 10bar",
    warranty: "2 года",
    },
    {
    id: "8",
    categoryId: "3",
    name: "CASIO PRG-650Y-1E",
    brand: 'Casio',
    price: 1050,
    image: "/uploads/9fdf7811068011e881af0050569f2b64_77a556710bb411e881af0050569f2b64-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "40 мм",
    glass: "минеральное",
    beltType: "полимер",
    faceColor: "черный",
    beltColor: "синий",
    waterProtection: "WR100 - 10bar",
    warranty: "2 года",
    },
    {
    id: "7",
    categoryId: "4",
    name: "FESTINA F16658/1",
    brand: 'Festina',
    price: 1219,
    image: "/uploads/f16658_1-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "40 мм",
    glass: "минеральное",
    beltType: "нержавеющая сталь",
    faceColor: "серебристый",
    beltColor: "серебристый",
    waterProtection: "WR100 - 10bar",
    warranty: "2 года",
    },
    {
    id: "6",
    categoryId: "4",
    name: "FESTINA F16659/2",
    brand: 'Festina',
    price: 853,
    image: "/uploads/f16659_2-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "44 мм",
    glass: "минеральное",
    beltType: "нержавеющая сталь",
    faceColor: "синий-фиолетовый",
    beltColor: "синий-фиолетовый",
    waterProtection: "WR100",
    warranty: "2 года",
    },
    {
    id: "4",
    categoryId: "5",
    name: "TISSOT T106.417.11.051.00",
    brand: 'Tissot',
    price: 1176,
    image: "/uploads/T106.417.11.051.00-720x720.png",
    gender: "мужские",
    type: "кварцевые",
    size: "44 мм",
    glass: "сапфировое",
    beltType: "нержавеющая сталь",
    faceColor: "черный",
    beltColor: "серебристый",
    waterProtection: "WR100 - 10bar",
    warranty: "2 года",
    },
    {
    id: "2",
    categoryId: "5",
    name: "TISSOT T099.405.11.418.00",
    brand: 'Tissot',
    price: 4731,
    image: "/uploads/T099.405.11.418.00-720x720.png",
    gender: "мужские",
    type: "механические",
    size: "42 мм",
    glass: "сапфировое",
    beltType: "нержавеющая сталь",
    faceColor: "прозрачный",
    beltColor: "серебристый",
    waterProtection: "WR50 - 5bar",
    warranty: "2 года",
    }
]
