// containers/watch/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import {Link} from 'react-router'

import {fetchWatchById, addWatchToBasket} from 'actions'
import {getWatchById} from 'selectors'
import BasketCart from 'components/basketCart'
import Header from 'components/header'
import Footer from 'components/footer'

class Watch extends Component{
    componentDidMount(){
        this.props.fetchWatchById(this.props.params.id)
    }

    renderFields(){
        const {watch} = this.props
       return(
           <div className='fields'>
               <h2>
                   {watch.name}
               </h2>
               <p>Бренд: {watch.brand}</p>               
               <p>Для кого: {watch.gender}</p>               
               <p>Тип механизма: {watch.type}</p>               
               <p>Размер циферблата: {watch.size}</p>               
               <p>Стекло: {watch.glass}</p>               
               <p>Материал ремня: {watch.beltType}</p>               
               <p>Цвет циферблата: {watch.faceColor}</p>               
               <p>Цвет браслета: {watch.beltColor}</p>               
               <p>Водозащита: {watch.waterProtection}</p>               
               <p>Гарантия: {watch.warranty}</p>               
           </div>
       )
    }

    renderContent(){
        const {watch} = this.props
        return(
            <div className='thumbnail'>
                <div className='row'>
                    <div className='col-md-6'>
                        <img 
                        className='img-thumbnail' 
                        src={watch.image} 
                        alt={watch.name}
                        />
                    </div>
                    <div className='col-md-6'>
                        {this.renderFields()}
                    </div>
                </div>
            </div>
        )
    }

    renderSidebar(){
        const {watch, addWatchToBasket} = this.props
        return(
            <div>
                <BasketCart className='productPageBasket'/>
                <div className='product_info'>
                    <h5>{watch.name}</h5>
                    <h5>{watch.price} рублей</h5>
                </div>                
                <button 
                className='btn_AddToCart'
                onClick={() => addWatchToBasket(watch.id)}
                >
                    Добавить в корзину
                </button>
                <Link to='/catalog' className='btn_toStore'>Вернуться в каталог</Link>
            </div>
        )
    }

    render(){
        console.log(this.props.watch)
        const {watch} = this.props
        return(
            <div className='view-container'>
            <Header/>
                <div className='container'>
                    <div className='row flex-row-reverse'>                        
                        <div className='col-md-3'>
                            {watch && this.renderSidebar()}
                        </div>
                        <div className='col-md-9'>
                            {watch && this.renderContent()}
                        </div>
                    </div>
                </div>
                <Footer />
            </div>
        )
    }
}

const mapStateToProps = state =>({
    watch: getWatchById(state, state.watchPage.id)
})

const mapDispatchToProps = {
    fetchWatchById,
    addWatchToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(Watch)
