// containers/watch/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import R from 'ramda'

import {fetchWatchById} from 'actions'
import {getWatchById} from 'selectors'

class Watch extends Component{
    componentDidMount(){
        this.props.fetchWatchById(this.props.params.id)
    }

    renderFields(){
        const {watch} = this.props
       return(
           <div>
               <h2>
                   {watch.name}
               </h2>
               <p>Бренд: {watch.brand}</p>               
               <p>Для кого: {watch.gender}</p>               
               <p>Тип механизма: {watch.type}</p>               
               <p>Размер циферблата: {watch.size}</p>               
               <p>Стекло: {watch.glass}</p>               
               <p>Материал ремня: {watch.beltType}</p>               
               <p>Цвет циферблата: {watch.faceColor}</p>               
               <p>Цвет браслета: {watch.beltColor}</p>               
               <p>Водозащита: {watch.waterProtection}</p>               
               <p>Гарантия: {watch.warranty}</p>               
           </div>
       )
    }

    renderContent(){
        const {watch} = this.props
        return(
            <div className='thumbnail'>
                <div className='row'>
                    <div className='col-md-6'>
                        <img 
                        className='img-thumbnail' 
                        src={watch.image} 
                        alt={watch.name}
                        />
                    </div>
                    <div className='col-md-6'>
                        {this.renderFields()}
                    </div>
                </div>
            </div>
        )
    }

    renderSidebar(){
        return(
            <div>
                Sidebar
            </div>
        )
    }

    render(){
        console.log(this.props.watch)
        const {watch} = this.props
        return(
            <div className='view-container'>
                <div className='container'>
                    <div className='row'>
                        <div className='col-md-9'>
                            {watch && this.renderContent()}
                        </div>
                        <div className='col-md-3'>
                            {watch && this.renderSidebar()}
                        </div>
                    </div>
                </div>
            </div>
        )
    }
}

const mapStateToProps = state =>({
    watch: getWatchById(state, state.watchPage.id)
})

const mapDispatchToProps = {
    fetchWatchById
}

export default connect(mapStateToProps, mapDispatchToProps)(Watch)
