// components/sliderMain/index.js
import React, { Component } from "react";
import Slider from "react-slick";

export default class SimpleSlider extends Component {
  render() {
    const settings = {
      dots: false,
      infinite: true,
      speed: 2000,
      autoplay: true,
      autoplaySpeed: 6000,
      slidesToShow: 1,
      slidesToScroll: 1,
      arrows: false
    };
    return (
      <div>
        <Slider {...settings}>
          <div>
            <div className="slider-content">
                <div className="slider-content-text">
                <h2>ROYAL LONDON 41334-04</h2>
                <p>Тип механизма: кварцевые</p>
                <p>Стекло: минеральное</p>
                <p>Материал ремня: натуральная кожа</p>
                <p>Цвет циферблата: белый</p>
                <p>Цвет браслета: коричневый</p>
                <p>Водозащита: WR50</p>
                <p>Гарантия: 2 года</p>
                <button>Перейти в каталог</button>
                </div>
                <div className="slider-content-image">
                    <img src="/uploads/41334-04-720x720.png" alt="Watch"/>
                </div>
            </div>
          </div>
          <div>
            <div className="slider-content">
                <div className="slider-content-text">
                <h2>CASIO EFS-S530L-2A</h2>
                <p>Тип механизма: кварцевые</p>
                <p>Стекло: сапфировое</p>
                <p>Материал ремня: натуральная кожа</p>
                <p>Цвет циферблата: синий</p>
                <p>Цвет браслета: синий</p>
                <p>Водозащита: WR100 - 10bar</p>
                <p>Гарантия: 2 года</p>
                <button>Перейти в каталог</button>
                </div>
                <div className="slider-content-image">
                    <img src="/uploads/a76590c7b1e111e881cb0050569f2b64_7da7dcd1baf711e881cb0050569f2b64-720x720.png" alt="Watch"/>
                </div>
            </div>
          </div>
          <div>
            <div className="slider-content">
                <div className="slider-content-text">
                <h2>FESTINA F16658/1</h2>
                <p>Тип механизма: кварцевые</p>
                <p>Стекло: минеральное</p>
                <p>Материал ремня: нержавеющая сталь</p>
                <p>Цвет циферблата: серебристый</p>
                <p>Цвет браслета: серебристый</p>
                <p>Водозащита: WR100 - 10bar</p>
                <p>Гарантия: 2 года</p>
                <button>Перейти в каталог</button>
                </div>
                <div className="slider-content-image">
                    <img src="/uploads/f16658_1-720x720.png" alt="Watch"/>
                </div>
            </div>
          </div>
          <div>
            <div className="slider-content">
                <div className="slider-content-text">
                <h2>TISSOT T106.417.11.051.00</h2>
                <p>Тип механизма: кварцевые</p>
                <p>Стекло: сапфировое</p>
                <p>Материал ремня: нержавеющая сталь</p>
                <p>Цвет циферблата: черный</p>
                <p>Цвет браслета: серебристый</p>
                <p>Водозащита: WR100 - 10bar</p>
                <p>Гарантия: 2 года</p>
                <button>Перейти в каталог</button>
                </div>
                <div className="slider-content-image">
                    <img src="/uploads/T106.417.11.051.00-720x720.png" alt="Watch"/>
                </div>
            </div>
          </div>
        </Slider>
      </div>
    );
  }
}
