// components/watchesForMainPage/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import {Link} from 'react-router'
import Slider from "react-slick";

import {
  fetchWatches, 
} from 'actions'
import {getWatches} from 'selectors'

class WatchesForMainPage extends Component {
  componentDidMount () {
    this.props.fetchWatches()
  }

  renderWatch(watch, index) {
    return(
      <div className="productSlider" key={index}>
          <img 
          src={watch.image} 
          alt={watch.name}
          />
          <div className="captionSlider">
            <h4>
              <Link to={`/watches/${watch.id}`}>{watch.name}</Link>
            </h4>
            <span className="priseSlider">
              <h4>{watch.price}руб.</h4>
              </span>
          </div>
      </div> 
    )
  }
  render() {
    const {watches} = this.props
    var settings = {
      dots: false,
      infinite: true,
      speed: 1000,
      autoplay: true,
      autoplaySpeed: 8000,
      slidesToShow: 5,
      slidesToScroll: 1,
      initialSlide: 0,
      responsive: [
        {
          breakpoint: 991,
          settings: {
            slidesToShow: 4,
            slidesToScroll: 4,
            infinite: false,
            dots: false,
            arrows: false
          }
        },
        {
          breakpoint: 575,
          settings: {
            slidesToShow: 3,
            slidesToScroll: 3
          }
        },
        {
          breakpoint: 480,
          settings: {
            slidesToShow: 2,
            slidesToScroll: 2
          }
        },
        {
          breakpoint: 370,
          settings: {
            slidesToShow: 1,
            slidesToScroll: 1
          }
        }
      ]
    };
    return (
      <div>
        <Slider {...settings}>
            {watches.map((watch, index) => <div>{this.renderWatch(watch, index)}</div>)}
        </Slider>
      </div>
    );
  }
}

const mapStateToProps = (state) => {
  return {
    watches: getWatches(state)
  }
}


const mapDispatchToProps = {
  fetchWatches
}

export default connect(mapStateToProps, mapDispatchToProps)(WatchesForMainPage)
